projector-client-web/src/main/kotlin/org/jetbrains/projector/client/web/window/OnScreenMessenger.kt

```diff
@@ -57,7 +58,7 @@ object OnScreenMessenger : LafListener {
     }
   }
 
-  private val text = (document.createElement("div") as HTMLDivElement).apply {
+  private val text = (document.createElement("p") as HTMLParagraphElement).apply {
     div.appendChild(this)
   }
 
```

Comment:
It looks more consistent
```suggestion
private val reload = (document.createElement("p") as HTMLParagraphElement).apply {
innerHTML = "If you wish, you can try to <a onclick='location.reload();' href=''>reconnect</a>."
```

Reply:
it's about next lines of code

Reply:
Let's leave it as is because it doesn't relate to the topic of the PR. Also, those lines are not changed here, and if it works, let's avoid changing it. It can be refactored later, in the scope of refactoring UI from plain DOM to something better like React or Compose.